### DISCUSSION TITLE
Need to remove html attribute aria-hidden

### DISCUSSION THREAD

**Discussion by sw-tt-jaydeepcharadva** [solspace/craft-freeform#1123] [Feature Requests]
Hello,

I want to remove  tab-index="-1" from my freeform form. Instead of tab-index="-1" I want to add disable attribute of html. Right now it is not allow me to add atribute in main div. I am attaching the screenshot of code for your reference. I need to do this because of Speed up my website as Site Improve suggested.

**Code View:**
![image](https://github.com/solspace/craft-freeform/assets/68269830/948a5b4d-8f17-41c4-a615-5b183afa93ce)

**Inspect Element view:**
![image](https://github.com/solspace/craft-freeform/assets/68269830/51dc02b2-f87b-4fc4-9fa9-20852668d4c5)

**SiteImprove Suggestions:**
![image](https://github.com/solspace/craft-freeform/assets/68269830/6aef4e5e-8311-43ee-9bc2-4fff81644a45)

**Comment by lukajincharadze** [comment]
By using the disabled attribute, you effectively disable all interactive elements within the <div> element, which would achieve the same result as setting tab-index="-1".

before:
<div id="myFreeform" aria-hidden="true" tabindex="-1">
  
</div>


after:
<div id="myFreeform" disabled>
  
</div>
